pkg/clusterapi/bottlerocket.go

```diff
@@ -85,3 +87,12 @@ func SetBottlerocketAdminContainerImageInKubeadmConfigTemplate(kct *bootstrapv1.
 func SetBottlerocketControlContainerImageInKubeadmConfigTemplate(kct *bootstrapv1.KubeadmConfigTemplate, versionsBundle *cluster.VersionsBundle) {
 	kct.Spec.Template.Spec.JoinConfiguration.BottlerocketControl = bottlerocketControl(versionsBundle.BottleRocketHostContainers.Control)
 }
+
+// SetBottlerocketInEtcdCluster adds bottlerocket config in etcdadmCluster.
+func SetBottlerocketInEtcdCluster(etcd *etcdv1.EtcdadmCluster, versionsBundle *cluster.VersionsBundle) {
```

Comment:
does this method need to be public or can we just test it through `EtcdadmCluster`?